unit-tests/test-dispatcher.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+let assert = require('chai').assert;
+let dispatcher = require('../lib/dispatcher/index.js');
+
+let STREAM = 'stream';
+
+// dispatcher unit tests
+describe('dispatcher', function() {
+
+  describe('constructor', function() {
+
+    var myDispatcher = new dispatcher(STREAM);
```

Comment:
Remove extra whitespace above declaration. Change variable name to `dispatcher`:

```js
let dispatcher = new Dispatcher(STREAM);
```